QUEST_LV_0100.tsv

```diff
@@ -1033,271 +1033,271 @@ QUEST_LV_0100_20150317_001032	$I should have requested this of you all at once.{
-QUEST_LV_0100_20150317_001227	Everything has a source called Mana. {nl} And everyone can connect to each other's Mana if they want. {nl} Just like digging a canal.
-QUEST_LV_0100_20150317_001228	$Have you ever seen Vubbes using pickaxes? {nl} I wonder what they are trying to find...
+QUEST_LV_0100_20150317_001225	$Still anxious. {nl}But thanks to you things are better than before.
+QUEST_LV_0100_20150317_001226	$I can't believe the Cunningham's story is true... I think it needs more research.
+QUEST_LV_0100_20150317_001227	$Everything has a source called Mana. {nl}And everyone can connect to each other's Mana if they want. {nl}As if unearthing a channel.
+QUEST_LV_0100_20150317_001228	$Have you ever seen Vubbes using pickaxes? {nl}I wonder what they are trying to find...
 QUEST_LV_0100_20150317_001229	$Thank you very much for saved us.{nl}
-QUEST_LV_0100_20150317_001230	$If you still want to enter the Closed Area, collect Vubbe Magic Stones from the Vubbes. {nl} And then insert them inside a gap and the Crystal Wall should open.
-QUEST_LV_0100_20150317_001231	$What are you doing here? {nl} Hurry and get into the mines. {nl} Weren't we supposed to meet in the 1st floor?
+QUEST_LV_0100_20150317_001230	$If you still want to enter the Closed Area, collect Vubbe Magic Stones from the Vubbes. {nl}And then insert them inside a gap and the Crystal Wall should open.
+QUEST_LV_0100_20150317_001231	$What are you doing here? Hurry and get into the mines. {nl}Weren't we supposed to meet in the 1st floor?
 QUEST_LV_0100_20150317_001232	$No demon will be able to pass here.{nl}I will do my best to protect this place so they can't interrupt the Paladin Master.
-QUEST_LV_0100_20150317_001233	Firm faith in us is the greatest shield.{nl}We always fought with our justice and faith.
+QUEST_LV_0100_20150317_001233	$Absolute faith is our greatest shield.{nl}In accordance with the Goddess's will, we came to fight using our justice and conviction.
 QUEST_LV_0100_20150317_001234	{memo X}Well done.{nl}As the Paladin Master, I salute to your accomplishment.
 QUEST_LV_0100_20150317_001235	$Watcher James
 QUEST_LV_0100_20150317_001236	$We must protect this place from the demons at any cost.{nl}The land here is sacred.
 QUEST_LV_0100_20150317_001237	$I would have fled long time ago if it wasn't for that pledge.{nl}I am still the son of the great Watcher.
 QUEST_LV_0100_20150317_001238	$We have to somehow thwart these demons.{nl}I cannot let this danger pass to our brothers in the church.
 QUEST_LV_0100_20150317_001239	{memo X}We can't go in due to the barrier.{nl}To open the door, we should go through 1st floor of the basement.
-QUEST_LV_0100_20150317_001240	When I meet you again, is this world going to be peaceful.. I hope it is.
-QUEST_LV_0100_20150317_001241	I never expected those demons to come down this far.{nl}I hope our brothers can persist in their fight against them.
+QUEST_LV_0100_20150317_001240	$Will this world be peaceful when we meet again... I hope it will.
```

Comment:
Will this world be peaceful when we meet again? 

(replace the ellipsis with a question mark?)